cmd/minikube/cmd/status.go

```diff
@@ -304,3 +362,141 @@ func statusJSON(st []*Status, w io.Writer) error {
 	_, err = w.Write(js)
 	return err
 }
+
+func readEventLog(name string) ([]cloudevents.Event, time.Time, error) {
+	path := localpath.EventLog(name)
+
+	st, err := os.Stat(path)
+	if err != nil {
+		return nil, time.Time{}, errors.Wrap(err, "stat")
+	}
+
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, st.ModTime(), errors.Wrap(err, "open")
+	}
+	var events []cloudevents.Event
+
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		ev := cloudevents.NewEvent()
+		if err = json.Unmarshal(scanner.Bytes(), &ev); err != nil {
+			return events, st.ModTime(), err
+		}
+		events = append(events, ev)
+	}
+
+	return events, st.ModTime(), nil
+}
+
+func clusterState(sts []*Status) ClusterState {
```

Comment:
this one could be moved to a better package in a follow up refactor ? I am ok with it being in cmd now

Reply:
Agreed. I initially had this as part of the `cluster` package, and the node status as part of the `node` package, but I brought it back to `status` to keep this PR small. There was an inverted dependency issue :|